src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -119,11 +123,20 @@
   [a b]
   (or (= a b) (some #{:any} [a b])))
 
+(defn- query-params-match?
```

Comment:
@camlow325 - I don't know if we have an accepted best practice around `defn-` right now.  I know we've waffled a bit on some projects (as I have personally).  And my observation has been that the Clojure community at-large does not exactly have its mind made up either - I see it used heavily in some projects and then not at all in others.